apps/shipments/models/shipment.py

```diff
@@ -486,6 +497,7 @@ def pick_up(self, document_id=None, asset_physical_id=None, **kwargs):
 
         if asset_physical_id:
             self.asset_physical_id = hashlib.sha256(asset_physical_id.encode()).hexdigest()
+            self.validate_gtx()
```

Comment:
My only concern with this being here instead of in a post_save signal is the increased RTT time for the pickup action.  @ajhodges thoughts?

Reply:
post_save still executes before the pickup action request returns - in order to have the request return prior to validation, we'd need to move it to a celery task. Not sure what the product requirements are, but it seems to me that we wouldn't want to validate the gtx tag after the fact.